benchmark/schema/blas-distributed.json

```diff
@@ -0,0 +1,70 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Distributed BLAS",
+  "description": "Input for a single BLAS operation with distributed vectors",
+  "type": "object",
+  "properties": {
+    "n": {
+      "type": "integer",
+      "minimum": 1
+    },
+    "r": {
+      "type": "integer",
+      "minimum": 1
+    },
+    "stride": {
+      "type": "integer",
+      "minimum": 1
+    },
+    "stride_x": {
+      "type": "integer",
+      "minimum": 1
+    },
+    "stride_y": {
+      "type": "integer",
+      "minimum": 1
+    }
+  },
+  "if": {
+    "required": [
+      "stride"
+    ]
+  },
```

Comment:
could you share the documentation for this?
I interpret it like if required stride then ...
but it never require stride right?